keras/models.py

```diff
@@ -240,14 +297,14 @@ def predict(self, X, batch_size=128, verbose=1):
                 preds = np.zeros(shape)
             preds[batch_start:batch_end] = batch_preds
 
-            if verbose == 1:
+            if verbose==1:
```

Comment:
PEP8: spaces around operators. We don't do strict PEP8, but such guidelines make the code more readable and should be respected.